DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/IdentityExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Security.Claims;
+using GitHubExtension.Notes.WebApi.ViewModels;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Notes.WebApi
+{
+    public static class IdentityExtensions
+    {
+        public static string GetUserId()
+        {
+            var userId = ClaimsPrincipal.Current.Identity.GetUserId();
+            return userId;
+        }
+
+        public static NoteModel AddUserIdToModel (this NoteModel noteModel)
+        {
+            var userId = ClaimsPrincipal.Current.Identity.GetUserId();
```

Comment:
Can you show me line number where UserId can be passed from UI? So next question, can I reuse your extension method?